setup.py

```diff
@@ -13,7 +13,7 @@
 url = "https://github.com/dabapps/django-forms-dynamic"
 author = "DabApps"
 author_email = "hello@dabapps.com"
-license = "BSD"
+license = "BSD-2-Clause"
```

Comment:
This is the old free-form license field, not the recent SPDX licence-expression field

Reply:
Ah, right. Good catch! It still passed the license to the `license` argument in `setup`. `pip show` shows `License-Expression` after changing the argument to `license_expression`.